deals/types.go

```diff
@@ -47,7 +47,7 @@ type StorageDealRecord struct {
 
 // RetrievalDealInfo contains information about a retrieval deal.
 type RetrievalDealInfo struct {
-	PieceCID                cid.Cid
+	RootCid                 cid.Cid
```

Comment:
No that I understand `PieceCID` is not the same as a data cid, this is the correct name